kratos/utilities/qef_utility.h

```diff
@@ -0,0 +1,157 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Ariadna Cortes
+//
+//
+
+#if !defined(KRATOS_QEF)
+#define  KRATOS_QEF
+
+// System includes
+
+// External includes
+
+// Project includes
+#include "containers/model.h"
+#include "includes/ublas_interface.h"
+#include "includes/geometrical_object.h"
+#include "includes/node.h"
+#include "geometries/geometry.h"
+#include "geometries/hexahedra_3d_8.h"
+#include "geometries/triangle_3d_3.h"
+#include "intersection_utilities.h"
+#include "../external_libraries/a_matrix/include/matrix.h"
```

Comment:
Why?, why don't use the Ublas matrix (the one everyone uses)

Reply:
Because i didnt know the difference :))))))